src/java.base/share/native/libjli/args.c

```diff
@@ -354,42 +354,38 @@ static JLI_List readArgFile(FILE *file) {
     return rv;
 }
 
+static void reportFcloseExit(FILE *fptr, const char* fmt, const char* arg) {
+    if (fmt != NULL) JLI_ReportMessage(fmt, arg);
+    if (fptr != NULL) fclose(fptr);
+    exit(1);
+}
```

Comment:
Can you insert a blank line here?

Reply:
This function is "optionally report, optionally fclose, and then exit". Have you tried reducing it to reportAndExit and fclose inline in expandArgFile to avoid it doing 3 things?

Reply:
> This function is "optionally report, optionally fclose, and then exit". Have you tried reducing it to reportAndExit and fclose inline in expandArgFile to avoid it doing 3 things?

hi Alan ,  thanks for your remark ,  I did not do that because I would copy both lines (fclose + exit)  to all callers.
On the other hand I wonder -  wouldn't  it be possible  to avoid the fclose  before the  exit(1)  ?
I assume the closing is done on exit anyway  ?

Reply:
That sounds okay to me, the process is exiting, as you say.